src/transformers/modeling_albert.py

```diff
@@ -327,6 +327,7 @@ def forward(
         hidden_states = self.embedding_hidden_mapping_in(hidden_states)
 
         all_attentions = ()
+        output_attentions = torch.tensor(output_attentions, dtype=torch.bool)
```

Comment:
This is not compatible with torch <= 1.2. Will find a way to make it compatible.

Reply:
Is our lib still compatible with torch <= 1.2 ? I feel like there are many cases where torch 1.2 does not work anymore

Reply:
If cast as a tensor of ints, is it breaking many things in the PR?